frontend/src/store/chats/reducer.ts

```diff
@@ -104,13 +104,31 @@ const reducer = createReducer(initialState, (builder) => {
 
   builder.addCase(createMessage.fulfilled, (state, { payload }) => {
     state.currentChatMessages = [...state.currentChatMessages, payload];
-    state.lastMessages = state.lastMessages.map((lastMessage) => {
-      if (lastMessage.chatId === payload.chatId) {
-        return payload;
-      }
 
-      return lastMessage;
-    });
+    const currentChatLastMessage = state.lastMessages.find(
+      (lastMessage) => lastMessage.chatId === payload.chatId,
+    );
+
+    if (currentChatLastMessage) {
+      state.lastMessages = state.lastMessages.map((lastMessage) => {
+        if (lastMessage.chatId === payload.chatId) {
+          return payload;
+        }
+
+        return lastMessage;
+      });
+    } else {
+      state.lastMessages = [...state.lastMessages, payload];
+      const notEmptyChat = state.emptyChats.find(
+        (emptyChat) => emptyChat.chatId === payload.chatId,
```

Comment:
But it seems like emptyChat does not have `chatId` field

Reply:
<img width="265" alt="image" src="https://user-images.githubusercontent.com/56650388/192029649-ab09db36-eefe-4ce1-80eb-66ad9d99bf80.png">
Do u mean this type? But emptyChat has chatId

Reply:
Missed that type :\